katas/langtonant/guillermo/module1.js

```diff
@@ -0,0 +1,81 @@
+const LEFT = 0;
+const UP = 1;
+const RIGHT = 2;
+const DOWN = 3;
+
+export class ant {
```

Comment:
For class name, lets use Pascal case so it can be distinguished from variables and functions